public/app/features/dashboard-scene/edit-pane/DummySelectedObject.tsx

```diff
@@ -0,0 +1,60 @@
+import { useMemo } from 'react';
+
+import { Input, TextArea } from '@grafana/ui';
+import { OptionsPaneCategoryDescriptor } from 'app/features/dashboard/components/PanelEditor/OptionsPaneCategoryDescriptor';
+import { OptionsPaneItemDescriptor } from 'app/features/dashboard/components/PanelEditor/OptionsPaneItemDescriptor';
+
+import { DashboardScene } from '../scene/DashboardScene';
+import { EditableDashboardElement } from '../scene/types';
+
+export class DummySelectedObject implements EditableDashboardElement {
```

Comment:
nit: I know there was a trend about avoiding using `dummy` when naming things. Maybe something like `SimpleSelectedObject` or `BasicSelectedObject`?

Reply:
Well this is just a temporary test object, so does not really matter what we call it. In the real PR this is implemented as a behavior attached to DashboardScene

Reply:
I missed the comment being temporary. Yeah, ignore this comment.